user_account/account_manager.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth.models import BaseUserManager
+
+
+class AccountManager(BaseUserManager):
+    def create_user(self, email, password, **extra_fields):
+        if not email:
+            raise ValueError("Email can't be empty")
+
+        user = self.model(email=self.normalize_email(email), **extra_fields)
+        user.set_password(password)
+        user.save()
+        return user
+
+    def create_superuser(self, email, password, **extra_fields):
+        extra_fields.setdefault("is_staff", True)
+        extra_fields.setdefault("is_superuser", True)
+        extra_fields.setdefault("is_active", True)
+        user = self.create_user(email, password, **extra_fields)
+        user.save()
+        return user
```

Comment:
```suggestion
        return self.create_user(email, password, **extra_fields)
```

you don't need to save this object again